### DISCUSSION TITLE
Google Cloud functions 2nd generation

### DISCUSSION THREAD

**Discussion by bachgarash** [serverless/serverless#11428] [Q&A]
Hi there.

I have a simple setup of GCP cloud functions deployed with Serverless framework. My question is how to activate 2nd generation of cloud functions from config .yml file. By default it deploys 1st gen of Cloud functions at the moment. 

Thanks in advance.

**Comment by pgrzesik** [comment]
Hey @Bakhtiyar-Garashov - at the moment this plugin does not support gen2 functions at all. 

@medikoo I can probably take a look at it in the coming weeks/months, I shared more details with you via Twitter, please let me know what do you think 🙇

**Comment by medikoo** [comment]
Great thanks @pgrzesik ! I've just provided you with needed rights to the repo 🙇

**Reply by pgrzesik** [reply]
thank you!
